HW03.rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "HW03"
+author: "Ben Wang"
+date: "7/20/20"
+output: github_document
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(dplyr)
+```
+
+### I decided to use the NYTimes coronavirus database to plot positive coronavirus cases in Illinois and New York, because these are the two places my brother and I are living. Below is the code for pulling the data from the New York Times database
+
+```{r}
+states <- read.csv(url("https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"))
+
+illinois <- states%>%
+  filter(state == "Illinois")
+nystate <- states%>%
+  filter(state == "New York")
+```
+
+
+```{r}
+###Created separate plot for Illinois
+illinois$date <- as.Date(illinois$date)
+illinois_plot <- ggplot(illinois, aes(date, cases/12671821 * 100)) + 
+  ###Population of IL estimated at 12,671,821 as of July 1st, 2020, by the US census bureau
```

Comment:
I like that you used the percentage of population which gives a much better comparison between two states rather than plotting the raw number of cases.  I think you would also be able to do this by using mutate( ).  With mutate you can create a new variable / column in the dataset that you could then call at any point in your code.

ex. illinois %>% mutate(PercentPopIL = cases/12671821 * 100)

I think this would then create a new variable called PercentPopIL that you could then reference.

Reply:
I agree, plotting the percentage of population is much more representative and easier for direct comparison. I never knew that it is possible for us to directly do calculations on a variable in the aes layer. If I was to do this, I would just assign the calculation to a new variable outside if the dataset and call it whenever I need it. This is what Ariana suggested but she also introduced a cool trick of making that new variable IN the dataset by using mutate(). This trick would allow for shorter and more straightforward way of writing the codes. I've learnt something new from the two of you, thank you!